CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Welcome!
+
+We're so glad you're thinking about contributing to a [open source project of the U.S. government](https://code.gov/)! If you're unsure about anything, just ask -- or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.
```

Comment:
Mention of the US Government seems wrong.... ;-)

Reply:
Sorry, I didn't mean to include the CONTRIBUTING.md file in the PR! That notice is required for my forked repository under the GSA organization.